--- A/CMakeLists.txt	2025-11-04 15:37:40.973142375 -0600
+++ B/CMakeLists.txt	2025-11-04 15:42:28.284732800 -0600
@@ -19,8 +19,9 @@
 find_package(cetmodules 3.24.01 REQUIRED)
 #project(icaruscode VERSION 10.06.00.06 LANGUAGES CXX)
 
-set(${PROJECT_NAME}_CMAKE_PROJECT_VERSION_STRING 10.06.00.06-p01)
 project(icaruscode LANGUAGES CXX)
+set(${PROJECT_NAME}_EXTENDED_VERSION_SEMANTICS True)
+set(${PROJECT_NAME}_CMAKE_PROJECT_VERSION_STRING 10.06.00.06-p01)
 
 message(STATUS
   "\n-- ============================================================================="
--- icaruscode-10_06_00_06p01/icaruscode/Overlays/CMakeLists.txt        2025-10-10 17:33:45.000000000 -0500
+++ spack-src/icaruscode/Overlays/CMakeLists.txt        2025-12-01 13:37:31.469405652 -0600
@@ -1,7 +1,7 @@
 find_package(gallery         REQUIRED)
 
 find_package(larwirecell REQUIRED )
-find_package(jsoncpp REQUIRED)
+find_package(jsoncpp REQUIRED COMPONENTS shared)
 find_package(spdlog REQUIRED)
 
 include_directories( $ENV{JSONCPP_INC} )
@@ -42,8 +42,8 @@
        WireCellAux
        WireCellIface
        WireCellPgraph
-       WireCellRoot
-       jsoncpp
+       spdlog::spdlog
+       jsoncpp_lib
        dl
 )

